modular_skyrat/modules/modular_slots/code/living.dm

```diff
@@ -0,0 +1,12 @@
+/// Gets the mob's current passport, if present. Purposefully doesn't support PDAs or wallets.
+/mob/living/proc/get_passport()
+	if(!length(held_items)) // Early return for mobs without hands.
+		return
+	// Check hands
+	var/obj/item/held_item = get_active_held_item()
+	if(held_item) // Check active hand
+		. = held_item.get_passport()
+	if(!.) // If there is no id, check the other hand
+		held_item = get_inactive_held_item()
+		if(held_item)
+			. = held_item.get_passport()
```

Comment:
```suggestion
			. = held_item.get_passport()
	return .
```
tg code standard

Reply:
Or better solution: Just use returns. There's no reason not to here.